src/Selenium2Library/__init__.py

```diff
@@ -19,7 +19,10 @@ class Selenium2Library(
     _WaitingKeywords,
     _AlertKeywords
 ):
-    """Selenium2Library is a web testing library for Robot Framework.
+    """Selenium2Library is a web testing library for Robot Framework. This
+	document is about using Selenium2Library. For information about
+	installation, support, and more please visit the
+	[https://github.com/robotframework/Selenium2Library|project page].
 
```

Comment:
thanks for the feedback. even though it is docstring, it exists to generate the library documentation, so I did not think about the normal conventions. I will update it.